### ISSUE TITLE
binarystore.xml is not copied on first startup

### ISSUE THREAD

**Issue by timdeluxe** [jfrog/charts#850]
**Is this a request for help?**:
No. (Or: Maybe?)

---

**Is this a BUG REPORT or FEATURE REQUEST?** (choose one):
BUG REPORT

**Version of Helm and Kubernetes**:
Helm 3.0.3 (via terraform helm provider 1.0.0)
Kubernetes v1.15.10

**Which chart**:
https://github.com/jfrog/charts/tree/master/stable/artifactory-oss

**What happened**:
binarystore.xml in /var/opt/jfrog/artifactory/etc/artifactory contains file-system configuration, while /artifactory_bootstrap/binarystore.xml contains azure-blob configuration.

**What you expected to happen**:
binarystore.xml from /artifactory_bootstrap/binarystore.xml gets copied on the first run to /var/opt/jfrog/artifactory/etc/artifactory as described in https://github.com/jfrog/charts/blame/7925230ef0f81c6a116dd9539cf3061f2ce52e3f/stable/artifactory/README.md#L414 (sorry, found no better way to link to a specific line in README.md without blame view)

**How to reproduce it** (as minimally and precisely as possible):
Setup a kubernetes cluster.
Apply chart with artifactory.persistence.type set to azure-blob (and needed other variables for that) - but i guess problem also exists with s3 storage etc.

**Anything else we need to know**:
I workarounded the problem by setting:
```
artifactory:
  artifactory:
    copyOnEveryStartup:
      - source: /artifactory_bootstrap/binarystore.xml
        target: etc/artifactory/
```

**Comment by chukka** [comment]
@timdeluxe  Thanks for reporting , Can you share the chart version ? does this happen only on artifactory-oss chart or artifactory as well ?

**Comment by timdeluxe** [comment]
@chukka 
I had version 2.1.0 before but also tried with 2.1.5 of the OSS chart. Didn't try 2.2.0 yet.
I only tried the artifactory-oss chart, since we did not buy a license.

Unfortunately i can not really troubleshoot anymore, because i found out, that the azure-blob storage feature is an artifactory pro feature and we use the OSS version. (I got strange stacktraces in the log, after implementing the workaround)
So we now live with the default, the file-system storage.
